website/docs/docs/deploy/state-aware-setup.md

```diff
@@ -216,7 +221,41 @@ With this config, dbt:
 
 If any new data is available _and_ at least 4 hours have passed, <Constant name="cloud" /> rebuilds the models.
 
+You can override freshness rules set at higher levels in your dbt project. For example, in the project file, you set:
+
+<File name="dbt_project.yml">
+```yml
+models:
+  +freshness:
+    build_after:
+      count: 4
+      period: hour
+  jaffle_shop:
```

Comment:
great info!

am i right in think this needs to match the project `name` in the dbt project.yml file? if so, might be good to call this out so its super clear?
```suggestion
  jaffle_shop: # this needs to match your project `name:` in dbt_project.yml
```